elfi/__init__.py

```diff
@@ -2,6 +2,7 @@
 from elfi.core import Transform, Simulator, Summary, Discrepancy
 from elfi.distributions import *
 from elfi.methods import *
+from elfi.post_proc import Result
```

Comment:
What about renaming this to e.g. `elfi.result`. ABC has these separate post processing methods which could cause confusion here.